pipeline/scraper/db.js

```diff
@@ -45,6 +45,44 @@ async function insertDev(dev) {
 }
 
 module.exports = {
+    /**
+     *  Query the search_terms table to get a list of terms that are stale
+     */
+    get_search_terms: async() => {
```

Comment:
@AdamSlack would prefer you reply in this thread :rofl: anyway yeah snake casing is the best :+1: but everybody seems to like camel casing here for some reason.